src/status_im/contexts/profile/push_notifications/effects.cljs

```diff
@@ -29,10 +29,9 @@
        (fn [_permissions]
          (pn-remote-token/request-fcm-token {})))
       (promesa/then
-       (fn [{:keys [_fcm-token]}]
+       (fn [{:keys [fcm-token]}]
          (logger/log-debug "registered fcm token for status news"
-                           ;; NOTE(@seanstrom): temporarily avoid logging fcm-token
-                           #_{:fcm-token fcm-token})))
+                           {:fcm-token fcm-token})))
```

Comment:
Here we're temporarily logging the Firebase token after successfully registering with Firebase for Status news notifications. This likely should not be logged in production, so this change has been marked as "donotmerge", but it still is required for testing, so we'll keep it in the PR for now.